File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/GIS/CoreWS_Gis.ts
import { APICallsBasic } from "./APICallsBasic";

export class CoreWS_Gis<T> extends APICallsBasic{
    public InData: T

    constructor (objP_super_server: string, objP_server: string, objP_utenti: string, inGisDataReadParam: T) {
        super(objP_super_server, objP_server, objP_utenti);
        this.InData = inGisDataReadParam;
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreWebServiceSTD/CoreWS_Gis.vb
Public Class CoreWS_Gis(Of T)
    Inherits APICallsBasic

    Public InData As T

    Public Sub New(ByVal objP_super_server As String, ByVal objP_server As String, ByVal objP_utenti As String, ByVal inGisDataReadParam As T)

        MyBase.New(objP_super_server, objP_server, objP_utenti)

        Me.InData = inGisDataReadParam

    End Sub

End Class
